front/hooks/useChangeMembersRoles.ts

```diff
@@ -0,0 +1,85 @@
+import type {
+  LightWorkspaceType,
+  RoleType,
+  UserTypeWithWorkspaces,
+} from "@dust-tt/types";
+import { useCallback, useContext } from "react";
+
+import { SendNotificationsContext } from "@app/components/sparkle/Notification";
+import { useMembers } from "@app/lib/swr/memberships";
+
+type HandleMembersRoleChangeParams = {
+  members: UserTypeWithWorkspaces[];
+  role: RoleType;
+};
+
+export function useChangeMembersRoles({
+  owner,
+}: {
+  owner: LightWorkspaceType;
+}) {
+  const sendNotification = useContext(SendNotificationsContext);
+  const { mutateMembers } = useMembers({
```

Comment:
don't forget to mutate search (a bug daph fixed this morning)

Reply:
Yes please 🙏🏻 😄